Userland/Libraries/LibIPC/Decoder.h

```diff
@@ -66,6 +68,17 @@ class Decoder {
         return true;
     }
 
+    template<Enum T>
+    bool decode(T& enum_value)
+    {
+        std::underlying_type_t<T> inner_value;
```

Comment:
You already have the import here, so you should just be able to do this:
```suggestion
        UnderlyingType<T> inner_value;
```

Reply:
Thanks Dex!